src/client/dfuse/dfuse_core.c

```diff
@@ -284,17 +221,12 @@ dfuse_start(struct dfuse_info *dfuse_info, struct dfuse_dfs *dfs)
 	fs_handle->dpi_max_read = 1024 * 1024 * 4;
 	fs_handle->dpi_max_write = 1024 * 1024;
 
-	rc = d_hash_table_create_inplace(D_HASH_FT_RWLOCK | D_HASH_FT_EPHEMERAL,
-					 3, fs_handle, &ie_hops,
+	rc = d_hash_table_create_inplace(D_HASH_FT_MUTEX | D_HASH_FT_LRU | D_HASH_FT_EPHEMERAL,
```

Comment:
we should really either enforce the 80 character rule where possible or change checkpatch to something like 120 and enforce that.

Reply:
Agree, I think this is a case where an exemption is warranted, but I think we should change it to 120.